libs/platform/src/lib/form/multi-combobox/commons/base-multi-combobox.ts

```diff
@@ -169,6 +173,15 @@ export abstract class BaseMultiCombobox extends CollectionBaseInput implements A
     @Input()
     invalidEntryMessage = 'Invalid entry';
 
+    /** Sets _limitless value */
+    @Input()
+    set limitless(val: boolean) {
```

Comment:
Setter just sets value to the private property, could be removed and `limitless` can be just an input

Reply:
done